src/webclients/components/dace/dace_panel.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020-2023 ETH Zurich and the DaCe-VSCode authors.
+// All rights reserved.
+
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import 'material-icons/iconfont/material-icons.css';
+
+import './dace_panel.css';
+
+import $ = require('jquery');
+(window as any).jQuery = $;
+
+import {
+    ICPCWebclientMessagingComponent
+} from '../../messaging/icpc_webclient_messaging_component';
+import { ComponentTarget } from '../../../components/components';
+import { ICPCRequest } from '../../../common/messaging/icpc_messaging_component';
+
+declare const vscode: any;
+
+class DaCePanel extends ICPCWebclientMessagingComponent {
+
+    private static readonly INSTANCE: DaCePanel = new DaCePanel();
+
+    private constructor() {
+        super(ComponentTarget.DaCe);
+    }
+
+    public static getInstance(): DaCePanel {
+        return this.INSTANCE;
+    }
+
+    private launchBtn?: JQuery<HTMLElement>;
+    private connectBtn?: JQuery<HTMLElement>;
+    private quitBtn?: JQuery<HTMLElement>;
+    private portInput?: JQuery<HTMLInputElement>;
+
+    public init(): void {
+        super.init(vscode, window);
+
+        this.launchBtn = $('#launch-btn');
+        this.connectBtn = $('#connect-btn');
+        this.quitBtn = $('#quit-btn');
+        this.portInput = $('#inputPort');
+
+        this.launchBtn.on('click', () => {
+            this.launchDaemon();
+        });
+        this.connectBtn.on('click', () => {
+            this.connectDaemon();
+        });
+        this.quitBtn.on('click', () => {
+            this.quitDaemon();
+        });
+
+        this.invoke('onReady');
+    }
+
+    private getPort(): number | undefined {
+        let port = this.portInput?.val();
```

Comment:
same as other comment - might be nice to configure this a-priori for some users

Reply:
Overrideable with settings, see above